### ISSUE TITLE
Overly lax speedrun achievements

### ISSUE THREAD

**Issue by WhenTanksFly** [ReactiveDrop/reactivedrop_public_src#827]
The idea is to reduce how often these speedruns are achieved in a public lobby _that wasn't even aiming for a speedrun_.

Tilarus - Communication Center: 3:21
----
I just scored 1:52 singleplayer with 2 marines total, and 2:03 solo. I suspect the current time is oriented towards [the map's very old version](https://steamcommunity.com/sharedfiles/filedetails/?id=1896506941) that had a few more vault doors and stuff on the way:
7.5s (flamer + 5 vindi alt) to 20.3s for gone corpse pile near start + 25.83s*2 for gone doors + ~2.2s for Wildcat flamering gone biomass right past the latter one of those doors...
That'd down the current requirement by 61.26-74.16s - 2:20 to 2:07. **2:20** sounds fine to me, keeping up with the idea that ~20s under req is possible in an efficient run.

----
Lana's Escape - Lana's Complex: 4:48
----
I can do 4:03 singleplayer with 2 marines total, and 4:34 solo. 2 main time savers are to call the evac ship early _(which will be simpler after #888 )_, and to split for first 2 north-east buttons _(after which I quickly ran back west through the closing gates of the middle/rail path)_.
If a hypothetical 2-man team only knows one of 2 key splits: 1 stays at buttons & 1 at finish -> 4:14, 2 buttons 0 finish -> 4:26. I'm tempted to call the latter variant a wrong approach, and want to see the requirement reduced to at least **4:25** - there's few sections where marine speed matters _(and a sizeable chunk of waiting for bomb is spent sitting behind the free turret)_, and most of obstacles on the bomb's way can be cleared very preemptively _(don't even need Adrenaline)_, so we give ~11s free time to the reasonable variant _(for comparison, to outheal a single Parasite with Faith Medgun takes about 8s, not requiring a complete stop)_.

----
Area 9800 - Power Plant's Generator: 2:25
----
Someone must've had **accidentally swapped** the times of Area 9800's middle missions: **Power Plant's Cooling Pump's 2:05** is way too hard - far from first try, I've barely managed it last second singleplayer with 3 marines total while avoding bot teleport/hyperspeeding, yet I can do Generator 41s below the req with the same team and rules. This aligns with my online speedrun experience of Generator being a mission where you never need to check on time, while on Cooling Pump even pros abuse bots.
Vice versa it would've been the usual ~20s free time for both.

----
Operation Cleansweep - Storage Facility: 1:05
----
Vegas Chainsaw + P-Rifle + Hornet Barrage (vrumvrum & rockets for doors while enemies are electricied by rifle alt) -> 0:51; Vegas Chainsaw + Flamer + Freeze -> 0:54; Crash P-Rifle x2 + Adrenaline (hacking both doors) -> 0:52; Sarge Heavy Rifle + Flamer + Damage Amplifier -> 0:49; Jaeger Ass. Rifle x2 + Hand Grenades -> 0:49... Wildcat can solo in 0:41 with hrifle+amp, clearing the path so others don't need to stop at all...
Such a short mission should be fine with a tighter requirement: **1:00**.

----
Bonus - AMBER Complex: 8:00
----
I don't agree with tuning it for botless solo _(just scored 7:38 first try after ditching tesla coils)_ despite many many splitting opportunities - it's not how speedruns are supposed to work, but I'm yet to see it achieved accidentally due to deaths, so maybe it's fine as is...
For non-solo, **6:30** still gives a huge room for errors _(my best time is 4:37 with many other players)_ yet sounds considerably less scary and demotivational. Maybe even 6:00 _(on leaderboards there's a few 5:00-5:30 times with me)_, though because of #886 I couldn't be bothered to do singleplayer duo yet.

**Comment by WhenTanksFly** [comment]
[merged into first post]
